transports/azure-storage-queues/configuration_config_asqn_[11,).partial.md

```diff
@@ -2,3 +2,5 @@
 Settings should be set when instantiating the transport:
 
 snippet: AzureStorageQueueConfigCodeOnly
+
+Use the [`QueueServiceClient`](https://learn.microsoft.com/en-us/dotnet/api/azure.storage.queues.queueserviceclient.-ctor?view=azure-dotnet), [`BlobServiceClient`](https://learn.microsoft.com/en-us/dotnet/api/azure.storage.blobs.blobserviceclient.-ctor?view=azure-dotnet) or [`TableServiceClient`](https://learn.microsoft.com/en-us/dotnet/api/azure.data.tables.tableserviceclient.-ctor?view=azure-dotnet) constructor overload that suits the authentication needs when creating the clients passed to the transport.
```

Comment:
Should we pulls this part into an .include and reuse?

Reply:
10 goes out of support soon and 8 has a different version. I was thinking it is not worth it.